lib/presentation/generators/views/uuid_generator_view.dart

```diff
@@ -0,0 +1,137 @@
+                      child: ListTile(
+                        title: Text("hyphens".tr),
+                      ),
+                      padding: const EdgeInsets.only(left: 8.0),
+                    ),
+                    actionWidget: Obx(
+                      () => Switch(
+                        onChanged: (value) => controller.hyphens.value = value,
+                        value: controller.hyphens.value,
+                      ),
+                    ),
+                  ),
+                  YaruRow(
+                    enabled: true,
+                    leadingWidget: FaIcon(FontAwesomeIcons.textHeight),
+                    trailingWidget: Padding(
+                      child: ListTile(title: Text("uppercase".tr)),
+                      padding: const EdgeInsets.only(left: 2.0),
+                    ),
+                    actionWidget: Obx(
+                      () => Switch(
+                        onChanged: (value) =>
+                            controller.uppercase.value = value,
+                        value: controller.uppercase.value,
+                      ),
+                    ),
+                  ),
+                  YaruRow(
+                    enabled: true,
+                    leadingWidget: FaIcon(FontAwesomeIcons.hashtag),
+                    trailingWidget: Padding(
+                      child: ListTile(title: Text("amount".tr)),
+                      padding: const EdgeInsets.only(left: 8.0),
+                    ),
+                    actionWidget: Flexible(
+                      child: Row(
+                        mainAxisAlignment: MainAxisAlignment.end,
+                        children: [
+                          Flexible(
+                            flex: 1,
+                            child: TextFormField(
+                              keyboardType: TextInputType.number,
+                              inputFormatters: [
+                                FilteringTextInputFormatter.digitsOnly
+                              ],
+                              textAlign: TextAlign.end,
+                              initialValue: controller.count.value.toString(),
+                              onChanged: (value) {
+                                controller.count.value = int.tryParse(value) ??
+                                    0; // e.g. if empty -> 0
+                              },
+                              decoration: InputDecoration(
+                                contentPadding: EdgeInsets.all(10),
+                                border: OutlineInputBorder(),
+                              ),
+                            ),
+                          ),
+                          const SizedBox(width: 8),
+                          Flexible(
+                            flex: 2,
+                            child: ElevatedButton(
+                              style: ElevatedButton.styleFrom(
+                                  backgroundColor: Colors.blue),
```

Comment:
I also removed this hard-coded Colors.blue ok? The color should inherit from the user selected theme in options.
![image](https://user-images.githubusercontent.com/52143624/195435790-3414c8c1-ec5a-443a-ad1a-2f5f8512a7d1.png)

Reply:
Alright thanks for showing me that function, I read something about it but wasn't sure.

I hardcoded the button as blue because it's supposed to be constrasting with the normal button (like in the original devtoys) but yeah this works fine too.

Great update!